### PR TITLE
Add case insensitive sorting

### PR DESCRIPTION
**Pull request by sasumner**
Implements #4479

### CODE REVIEW

File: PowerEditor/src/MISC/Common/Sorters.h [hunk 1]
```diff
@@ -130,6 +130,49 @@ class LexicographicSorter : public ISorter
...
 
+// Implementation of lexicographic sorting of lines, ignoring character casing
+class LexicographicCaseInsensitiveSorter : public ISorter
+{
+public:
+	LexicographicCaseInsensitiveSorter(bool isDescending, size_t fromColumn, size_t toColumn) : ISorter(isDescending, fromColumn, toColumn) { };
+
+	std::vector<generic_string> sort(std::vector<generic_string> lines) override
+	{
+		// Note that both branches here are equivalent in the sense that they give always give the same answer.
```
Comment (chcg) [inline PowerEditor/src/MISC/Common/Sorters.h:141]: "they give always give" seems to be one give to much

Comment (sasumner) [reply PowerEditor/src/MISC/Common/Sorters.h:141]: If you search the code for those four words BEFORE this PR, you will find them.
Thus, this code is blameless for that, as it is a copy+paste. :-)
But, I will fix both.

**Comment by sasumner** [comment]
@donho I was expecting more conversation about this rather than acceptance with no changes. :-)  We will see what the users think of it.

**Comment by donho** [comment]
@sasumner 
What kind of point are we supposed to talk about ?

**Comment by sasumner** [comment]
@donho 

Nothing specific.
Maybe I am just too worried about this one. :-)

**Pull request closed without merging**